toolkit/scripts/toolchain/create_toolchain_in_container.sh

```diff
@@ -12,7 +12,17 @@ INCREMENTAL_TOOLCHAIN=$4
 ARCHIVE_TOOL=$5
 
 # Grab an identity for the raw toolchain components so we can avoid rebuilding it if it hasn't changed
-sha_component_tag=$(sha256sum ./container/toolchain-sha256sums ./container/toolchain_build_in_chroot.sh ./create_toolchain_in_container.sh | sha256sum | cut -d' ' -f1 )
+files_to_watch=( "./create_toolchain_in_container.sh" \
+                    "./container/toolchain-sha256sums" \
```

Comment:
Can you also add `toolchain-remote-wget-list`?

Reply:
and please re-alphabetize the list

Reply:
done